webextension/background.js

```diff
@@ -289,6 +289,44 @@ const backgroundLogic = {
     });
   },
 
+  openTab(options) {
+    const url = options.url || undefined;
+    const userContextId = ("userContextId" in options) ? options.userContextId : 0;
+    const active = ("nofocus" in options) ? options.nofocus : true;
+/*
+TODO check if we are opening in the correct window
+    return new Promise(resolve => {
+      if ("window" in args && args.window) {
+        resolve(args.window);
```

Comment:
Need to check if the window finding code behaves the same, currently tab.create seems to behave the same.